pkg/apis/serving/v1beta1/predictor_torchserve.go

```diff
@@ -86,7 +110,22 @@ func (t *TorchServeSpec) GetContainer(metadata metav1.ObjectMeta, extensions *Co
 		arguments = append(arguments, fmt.Sprintf("%s=%s", constants.ArgumentWorkers, strconv.FormatInt(*extensions.ContainerConcurrency, 10)))
 	}
 	if t.Container.Image == "" {
-		t.Container.Image = config.Predictors.PyTorch.ContainerImage + ":" + *t.RuntimeVersion
+		t.Container.Image = config.Predictors.PyTorch.V1.ContainerImage + ":" + *t.RuntimeVersion
+	}
+	t.Name = constants.InferenceServiceContainerName
+	t.Args = arguments
+	return &t.Container
+}
+
+func (t *TorchServeSpec) GetContainerV2(metadata metav1.ObjectMeta, extensions *ComponentExtensionSpec, config *InferenceServicesConfig) *v1.Container {
+	arguments := []string{
+		"torchserve",
+		"--start",
+		fmt.Sprintf("%s=%s", "--model-store", constants.DefaultModelLocalMountPath+"/model-store"),
```

Comment:
is `model-store` subdir  required and torchserve always expects this?

Reply:
No, we are following that folder structure to have config and mar files seperated